components-core/src/main/java/com/adyen/checkout/components/core/internal/data/api/AnalyticsMapper.kt

```diff
@@ -75,22 +76,22 @@ class AnalyticsMapper {
         private const val DROP_IN_COMPONENT = "dropin"
         private const val ANDROID_CHANNEL = "android"
 
-        private var PLATFORM = AnalyticsPlatform.ANDROID.value
-        private var VERSION = BuildConfig.CHECKOUT_VERSION
+        private var platform = AnalyticsPlatform.ANDROID.value
```

Comment:
@OscarSpruit let me know if you suppress the warning on these variable or maybe rename them to indicate that they are technically constants.

Reply:
> Names of top-level or object properties which hold objects with behavior or mutable data should use camel case names:

Taken from [the official Kotlin conventions guide](https://kotlinlang.org/docs/coding-conventions.html#property-names). So maybe we just accept the lowercase naming? Or we rename them?

Reply:
What if we keep them camel case but name them something more descriptive? Like `actualPlatform` and `actualVersion`? I can't think of a good name :/

Reply:
`actual...` sounds good to me. It makes it a bit more clear imo